Lib/fontTools/ttLib/tables/_p_o_s_t.py

```diff
@@ -252,14 +255,34 @@ def fromXML(self, name, attrs, content, ttFont):
 			self.data = readHex(content)
 
 
-def unpackPStrings(data):
+def unpackPStrings(data, n):
+        # extract n Pascal strings from data.
+        # if there is not enough data, use ""
+
 	strings = []
 	index = 0
 	dataLen = len(data)
-	while index < dataLen:
-		length = byteord(data[index])
-		strings.append(tostr(data[index+1:index+1+length], encoding="latin1"))
-		index = index + 1 + length
+
+	for _ in range(n):
+		if (dataLen <= index):
```

Comment:
`if` statements in python don't need the parentheses, they are no-op

Reply:
Fixed.